tools/UpdateDotnetRuntime.ps1

```diff
@@ -356,3 +361,6 @@ if ($dotnetUpdate.ShouldUpdate) {
 
     Update-DevContainer
 }
+else {
+    Write-Verbose -Verbose -Message $dotnetUpdate.Message
```

Comment:
```suggestion
    Write-Verbose -Verbose -Message "No update needed.  DotNet update message: $($dotnetUpdate.Message)"
```

Reply:
an update may actually be needed, so i wouldn't want to say "update needed" when the update message says "update manually"